pesde.toml

```diff
@@ -31,7 +30,7 @@ lib = "lib/init.luau"
 default = "https://github.com/pesde-pkg/index"
 
 [dev_dependencies]
-stylua = { name = "pesde/stylua", version = "=2.0.2", target = "lune" }
-frktest = { name = "itsfrank/frktest", version = "^0.0.2", target = "lune" }
-asciitable = { name = "kimpure/asciitable", version = "^0.2.1" }
-luau-lsp = { name = "pesde/luau_lsp", version = "=1.39.2", target = "lune" }
+stylua = { name = "pesde/stylua", version = "=2.1.0", target = "lune" }
+frktest = { name = "abidbmt/frktest", version = "^0.1.1", target = "lune" }
```

Comment:
We should probably wait for https://github.com/itsfrank/frktest/pull/9 to be merged before I merge this. Using a fork I have no control over seems like a supply-chain attack waiting to happen.